Update list of countries with restricted Cloud access [HC-1571] (#388)
arduino/help-center-content: content/Arduino Cloud/General Cloud/Availability-of-Arduino-Cloud-in-different-countries.md

* Sudan
* Syria
* Ukraine
* United Arab Emirates
* Zimbabwe

Specifically, these restrictions apply:

* You will be unable to purchase an Arduino Cloud plan.

> * Because some ancillary services used by Arduino Cloud may block access to these countries, using Arduino Cloud may not be possible even with the Free plan.

Reviewer: Thought the second bullet point was hard to understand. Might just be me tho :) 

```suggestion
* Due to potential restrictions from ancillary services used by Arduino Cloud, accessing Arduino Cloud may not be possible even with the Free plan.
```
